build.ps1

```diff
@@ -294,4 +295,116 @@ if ($Test) {
     Invoke-Pester -ErrorAction Stop
 }
 
+if ($Msix) {
+    if (!$IsWindows) {
+        throw "MSIX is only supported on Windows"
+    }
+
+    if ($architecture -eq 'current') {
+        throw 'MSIX requires a specific architecture'
+    }
+
+    $makeappx = Get-Command makeappx -CommandType Application -ErrorAction Ignore
+    if ($null -eq $makeappx) {
+        # try to find
+        if ($architecture -eq 'aarch64-pc-windows-msvc') {
+            $arch = 'arm64'
+        }
+        else {
+            $arch = 'x64'
+        }
+
+        $makeappx = Get-ChildItem -Recurse -Path (Join-Path ${env:ProgramFiles(x86)} 'Windows Kits\10\bin\*\' $arch) -Filter makeappx.exe | Sort-Object FullName -Descending | Select-Object -First 1
+        if ($null -eq $makeappx) {
+            throw "makeappx not found"
```

Comment:
It would help if there was a download link or instruction here how to get makeappx.

Reply:
Fair point, I'll update the text, but it's basically installing Windows SDK